test/main/udf_test.cpp

```diff
@@ -19,6 +29,22 @@ TEST_F(ApiTest, UnaryUDFInt64) {
     sortAndCheckTestResults(actualResult, expectedResult);
 }
 
+TEST_F(ApiTest, TestDropUDF) {
+    conn->createScalarFunction("add5", &add5);
+    conn->removeUDFFunction("add5");
+    conn->createScalarFunction("add5", &add5);
+}
+
+TEST_F(ApiTest, TestExceptUDF) {
+    conn->createScalarFunction("test", &throwRuntimeException);
+    try {
+        conn->query("RETURN test()");
+    } catch (RuntimeException) {}
+    conn->removeUDFFunction("test");
+    conn->createScalarFunction("test", &dontThrowRuntimeException);
+    conn->query("RETURN test()");
+}
+
```

Comment:
You mentioned in the description that:
We can now create a zero parameter udf function. Can you add a test for it?